app/controllers/spree/admin/taxjar_settings_controller.rb

```diff
@@ -1,17 +1,17 @@
 module Spree
   module Admin
     class TaxjarSettingsController < Spree::Admin::BaseController
+      helper_method :taxjar_tax_category_option
+
       def edit
         @configuration = SuperGood::SolidusTaxjar.configuration
         @nexus_regions = Rails.cache.fetch(:nexus_regions) || []
+        @tax_categories = cache_tax_categories
```

Comment:
If we have a button to pull down the tax categories on demand, do we still want to try and do it on load as well?